src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -16,20 +19,38 @@
 
 public class SearchConceptsQueryBuilder {
 
+  // look up domainID in domainOptions
+
   private SearchConceptsQueryBuilder() {}
 
   public static String buildSearchConceptsQuery(
-      String domainId,
+      SnapshotBuilderDomainOption domainOption,
       String searchText,
       TableNameGenerator tableNameGenerator,
       CloudPlatformWrapper platform) {
     var conceptTablePointer = TablePointer.fromTableName("concept", tableNameGenerator);
+    var occurrenceTable = getOccurrenceTableFromDomain(domainOption.getId());
+    var domainOccurrencePointer =
+        TablePointer.fromTableName(occurrenceTable.tableName(), tableNameGenerator);
     var conceptTableVariable = TableVariable.forPrimary(conceptTablePointer);
     var nameField = conceptTableVariable.makeFieldVariable("concept_name");
     var idField = conceptTableVariable.makeFieldVariable("concept_id");
 
+    // FROM concept JOIN conditionOccurrencePointer ON conditionOccurrencePointer.concept_id =
+    // concept.concept_id
+    var conditionOccurenceTableVariable =
```

Comment:
nit: naming in comment is outdated and naming of conditionOccurenceTableVariable

Reply:
This has been updated !